app/pods/controller-injections-2/child-route/controller.js

```diff
@@ -1,9 +1,11 @@
 import Controller from '@ember/controller';
+import { inject as controller } from '@ember/controller';
 import { computed } from "@ember/object";
 
 export default Controller.extend({
+  parentController: controller('controller-injections-2'),
 
-  dataOriginal: null, // Should be populated from parent route
+  dataOriginal: computed.alias("parentController.data"), // Should be populated from parent route
```

Comment:
`alias` lets you overwrite the property, and Ember two-way binding will propagate this change to the parent.

There are legit cases to this, but most of the time you only need to read values. You can protect such values from being rewritten by using `readOnly` instead of `alias`.